frontend-project/src/locales/en-US/menu.ts

```diff
@@ -17,6 +17,8 @@ export default {
   'menu.institutions': 'Institutions',
   'menu.institutions.new': 'New',
   'menu.letters': 'Letters',
+  'menu.letters.list': 'List',
```

Comment:
z kontekstu zgaduję, że Letter list by to lepiej opisywało

Reply:
jest to dropdown menu, gdzie we wszystkich innych przypadkach używamy 'List'